packages/base-types/src/request/index.ts

```diff
@@ -7,6 +7,7 @@ export enum RequestType {
   TEXT = 'text',
   ACTION = 'action',
   INTENT = 'intent',
+  EVENT = 'event',
```

Comment:
Should we perhaps name this as `ALEXA_EVENT = 'alexa_event'`? 

My concern is that I think we have a Voiceflow concept also called "Event" that we're planning to implement in the future, but I haven't been caught up in the discussion for that. I *think* it refers to a very generic request payload that we send to the DM API, which is used to trigger a Choice block. The difference between the VF Event payload and an Intent is that an Event could be triggered by anything that's not an utterance, e.g, a button click on a website. 

In keep the naming clear, we might not want to name this Alexa-specific data as "event" and then have to name our event system as "vf_event".

Reply:
refactored so we dont need this anymore :)